CMakeLists.txt

```diff
@@ -16,23 +16,51 @@ else()
   message(FATAL_ERROR "UTOPIA_HLS_HOME env var is not set!")
 endif()
 
-find_package(MLIR REQUIRED CONFIG)
-set_property(TARGET LLVMSupport APPEND
-  PROPERTY INTERFACE_INCLUDE_DIRECTORIES "${LLVM_INCLUDE_DIRS}")
-set_property(TARGET MLIRSupport APPEND
-  PROPERTY INTERFACE_INCLUDE_DIRECTORIES "${MLIR_INCLUDE_DIRS}")
-
 include(UtopiaFindHelper)
 set_local_from_env(CTemplate_ROOT CT_DIR)
 set_local_from_env(XercesC_ROOT XERCES_DIR)
```

Comment:
Are we still need this? Xerces lib is installed as package (no build from sources).

Reply:
Yes. In this project i have ip-xact parser, which needs it.